entities.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+class Position(object):
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def move(self, x, y):
+        self.x += x
+        self.y += y
+
+    def __repr__(self):
+        return "(%d, %d)" % (self.x, self.y)
+
+
+STARTING_PLAYER_POS = Position(0, 0)
+
+
+class ProgrammingError(Exception):
+    pass
+
+
+class Entity(object):
+
+    _pool = []
```

Comment:
Maybe it'd be better to let a map object handle its own entity and character pools?